_field-types/mapping-parameters/term-vector.md

```diff
@@ -0,0 +1,260 @@
+  "positions": true,
+  "offsets": true
+}
+```
+{% include copy-curl.html %}
+
+Expected result:
+
+```json
+{
+  "_index": "articles",
+  "_id": "1",
+  "_version": 1,
+  "found": true,
+  "took": 4,
+  "term_vectors": {
+    "content": {
+      "field_statistics": {
+        "sum_doc_freq": 9,
+        "doc_count": 1,
+        "sum_ttf": 9
+      },
+      "terms": {
+        "an": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 2,
+              "start_offset": 14,
+              "end_offset": 16
+            }
+          ]
+        },
+        "analytics": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 7,
+              "start_offset": 40,
+              "end_offset": 49
+            }
+          ]
+        },
+        "and": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 6,
+              "start_offset": 36,
+              "end_offset": 39
+            }
+          ]
+        },
+        "is": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 1,
+              "start_offset": 11,
+              "end_offset": 13
+            }
+          ]
+        },
+        "open": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 3,
+              "start_offset": 17,
+              "end_offset": 21
+            }
+          ]
+        },
+        "opensearch": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 0,
+              "start_offset": 0,
+              "end_offset": 10
+            }
+          ]
+        },
+        "search": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 5,
+              "start_offset": 29,
+              "end_offset": 35
+            }
+          ]
+        },
+        "source": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 4,
+              "start_offset": 22,
+              "end_offset": 28
+            }
+          ]
+        },
+        "suite": {
+          "doc_freq": 1,
+          "ttf": 1,
+          "term_freq": 1,
+          "tokens": [
+            {
+              "position": 8,
+              "start_offset": 50,
+              "end_offset": 55
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+## Highlighting with term vectors
+
+Use the following command to search for the term "analytics" and highlight it using the field's stored term vectors:
```

Comment:
Please link to the highlighting documentation for more info. https://docs.opensearch.org/docs/latest/search-plugins/searching-data/highlight/